app/parsers/bulkrax/csv_parser_decorator.rb

```diff
@@ -0,0 +1,146 @@
+# frozen_string_literal: true
```

Comment:
these both seem like bulkrax changes and not overrides to me.

Reply:
This is temporarily being added for the migration due to their custom files. If some of it is desired behavior, it could be contributed back to bulkrax, but the intent here was just to be able to handle the mobius custom files.

Reply:
We talked about pushing the `untar` method back to Bulkrax, but I agree that the "combined" logic could (and should) go back as well. 

@kirkkwang if asked, how quickly do you think you could port the `#untar` and combined logic to a Bulkrax branch and point HykuUP Knapsack at it?